src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/settingsComponents/ScoringCard.jsx

```diff
@@ -14,11 +14,19 @@ export const ScoringCard = ({
 }) => {
   const { handleMaxAttemptChange, handleWeightChange } = scoringCardHooks(scoring, updateSettings);
 
+  const getScoringSummary = (attempts, unlimited, weight) => {
+    let summary = unlimited
+      ? intl.formatMessage(messages.unlimitedAttemptsSummary)
+      : intl.formatMessage(messages.attemptsSummary, { attempts });
+    summary += ` ${String.fromCharCode(183)} `;
+    summary += intl.formatMessage(messages.weightSummary, { weight });
+    return summary;
+  };
+
   return (
     <SettingsOption
       title={intl.formatMessage(messages.scoringSettingsTitle)}
-      summary={intl.formatMessage(messages.scoringSummary,
-        { attempts: scoring.attempts.number, weight: scoring.weight })}
+      summary={getScoringSummary(scoring.attempts.number, scoring.attempts.unlimited, scoring.weight)}
     >
       <Form.Group>
```

Comment:
<img width="258" alt="Screen Shot 2023-01-20 at 11 07 40 AM" src="https://user-images.githubusercontent.com/42981026/213747265-19f0937c-9e43-4148-9e65-527270543c21.png">
need to add this message here